src/BlockComponent.vue

```diff
@@ -0,0 +1,39 @@
+<script setup lang="ts">
+import { computed } from "vue";
+import { FileContext, FolderContext, Block } from "@githubnext/blocks";
+
+const { context, block } = defineProps<{
+  context: FileContext | FolderContext;
+  block: Block;
+}>();
+
+const hash = computed(() =>
+  encodeURIComponent(
+    JSON.stringify({
+      block: {
+        owner: block.owner,
+        repo: block.repo,
+        id: block.id,
+        type: block.type,
+      },
+      context,
+    })
+  )
+);
+</script>
+
+<template>
+  <iframe
+    title="embedded block"
+    :src="`/#${hash}`"
```

Comment:
Since you're already using a computed property here, why not move all of the string logic into _that_ function, resulting in a cleaner:

```vue
:src="hash"
```